shm/src/cargo.cpp

```diff
@@ -1,4 +1,4 @@
 #include "cargo.hpp"
 
-Cargo::Cargo(const std::string& name, size_t basePrice, size_t amount)
+Cargo::Cargo(const std::string& name, size_t amount, size_t basePrice)
```

Comment:
On discord you said that you changed order because initialisation was not correct. Do you know why?